autograder/autograder/execution_environments/secure_execution_environment.py

```diff
@@ -324,3 +324,8 @@ def log_stack_trace(self, trace):
                                       is_batch = self.is_batch,
                                       which_untrusted = self.untrusted_user,
                                       trace = trace)
+
+  """ A useful wrapper for the atuograding_utils.log_message function. """
+  def log_container_meta(self, name, container, event, time):
```

Comment:
You can shorten several of your function calls by re-arranging the parameter list and adding defaults
```
(self, event, name='', container='', time=0)
```

Reply:
Changed.